src/convertObjectKeyIntoLabel/index.js

```diff
@@ -0,0 +1,6 @@
+const convertObjectKeyIntoLabel = (objectKey) => {
```

Comment:
@scottdougan @ahmedpanju 
I feel this is more generic than just object keys how do you feel about the name `snakeCasetoReadableString`?

Reply:
For reference in common we have
```js
function labelFromKey(key) {
  return (key || '').split('_')
    .map(element => element.charAt(0) + element.slice(1).toLowerCase())
    .join(' ');
}
```

Reply:
Incorporate upperCase change and make the key || "" a default prop